modules/stochastic_tools/doc/content/modules/stochastic_tools/examples/gaussian_process_surrogate.md

```diff
@@ -0,0 +1,105 @@
+
+!equation
+\begin{aligned}
+T(x,k,q,L,T_{\infty}) &= \frac{q}{2k}\left(L^2 - x^2\right) + T_{\infty} \\
+\bar{T}(k,q,L,T_{\infty}) &= \frac{qL^2}{3k} + T_{\infty} \\
+\end{aligned}
+
+For that reason $\bar{T}$ is chosen as the QoI to be modeled by the surrogate for this example problem. The closed form solution allows for easy comparison between the surrogate and the exact solution. 
+
+
+## Setting up a 1D Problem
+
+Problems with a single input variable are a good place to provide insight on Gaussian Process regression. To accomplish this three parameters of our model system are fixed $\lbrace k=5, L=0.03, T_{\infty}=300 \rbrace$, leaving the surrogate to only model the action of varying $q$.
+
+6 training values for $q$ were selected from $\sim\mathcal{U}(1, 10)$ and evaluated using a full model evaluation. The Gaussian Process model was fitted to these data points.
+
+The Gaussian Process was chosen to use a [SquaredExponentialCovariance.md] covariance function, using three user selected hyperparameter settings: $\lbrace \sigma_n=1E-3, \sigma_f=1 , \ell_q=0.38971 \rbrace$. To set up the training for this surrogate we require the standard `Trainers` block found in all surrogate models in addition to the Gaussian Process specific `Covariance` block. Hyperparameters vary depending on the covariance function selected, and are therefor specified in the `Covariance` block.
+
+!listing examples/surrogates/gaussian_process/gaussian_process_uniform_1D.i block=Trainers Covariance
+
+Creation of the `Surrogate` block follows the standard procedure laid out for other surrogate models.
+
+!listing examples/surrogates/gaussian_process/gaussian_process_uniform_1D.i block=Surrogates
+
+One advantage of Gaussian Process surrogates is the ability to provide an model for uncertainty. To output this data the standard `EvaluateSurrogate` vector post processor is replaced with the `EvaluateGaussianProcess` vector post processor, which functions similarly but also outputs the standard deviation of the surrogate at the evaluation point.  
+
+!listing examples/surrogates/gaussian_process/gaussian_process_uniform_1D.i block=VectorPostprocessors
+
+The Gaussian Process surrogate model can only be evaluated at discrete points in the parameter space, so if we wish to visualize the response model fine sampling is required. To accomplish this sampling a `CartesianProduct` sampler evaluates the model for 100 evenly spaced $q$ values in $[9000,11000]$. This sampling is plotted below in [1D_untuned] (space between the 100 sampled points are filled by simple linear interpolation, so strictly speaking the plot is not exactly the model)
+
+!listing examples/surrogates/gaussian_process/gaussian_process_uniform_1D.i block=Samplers
+
+!media gaussian_process_uniform_1D.svg id=1D_untuned caption=Surrogate model for 1D problem $\bar{T}(q)$ using fixed (user specified) hyperparameters.
```

Comment:
These figures are pretty big. You can make them smaller by doing `style=width:60%`. But I will leave this to your discretion.